code/modules/fallout/obj/items.dm

```diff
@@ -26,7 +26,7 @@
 		var/mob/living/carbon/M = loc
 		if(src == M.back)
 			if(M.health < M.maxHealth)
-				M.adjustBruteLoss(-2) //Heal that poor bastard
-				M.adjustFireLoss(-2)
-				M.adjustToxLoss(-2)
-				M.adjustOxyLoss(-2)
+				M.adjustBruteLoss(-5) //Heal that poor bastard
+				M.adjustFireLoss(-3)
+				M.adjustToxLoss(-2) 
+				M.adjustOxyLoss(-2) //Why do we need oxy healing?
```

Comment:
bloodloss causes oxy damage. you can up this to like ... 5 or 10 to be effective